kaiax/auction/impl/bid_pool.go

```diff
@@ -219,7 +220,8 @@ func (bp *BidPool) AddBid(bid *auction.Bid) (common.Hash, error) {
 	bp.bidMu.Lock()
 	defer bp.bidMu.Unlock()
 
-	if len(bp.bidMap) >= MaxBidNum {
+	if len(bp.bidMap) >= int(bp.maxBidPoolSize) {
+		logger.Info("Bid pool is full", "maxBidPoolSize", bp.maxBidPoolSize, "bid", bid.Hash())
```

Comment:
How about adding a metric when a bid is inserted?

Reply:
Could you recommend metrics? I can think of something like `numBidsGauge`.

Reply:
Thanks. [9fd1bf9](https://github.com/kaiachain/kaia/pull/466/commits/9fd1bf966f83f3009c7de14e856920a75ec06f47)